s64da_benchmark_toolkit/netdata.py

```diff
@@ -86,12 +86,12 @@ def _write_stats_no_breakdown(self, df, output):
         netdata_df = self.get_system_stats(df, 1)
 
         with open(output, 'w') as output_file:
-            output_file.write(f'all')
+            output_file.write(f'all\n')
             netdata_df.to_csv(output_file)
             output_file.write('\n')
 
     def write_stats(self, df, output):
         if len(df['stream_id'].unique()) == 1:
             self._write_stats_per_query(df, output)
-        else:
-            self._write_stats_no_breakdown(df, output)
+
+        self._write_stats_no_breakdown(df, output)
```

Comment:
As per query data was already there, I decided to keep it. Maybe SE needs it. But that's ok, we can parse it.

Reply:
But originally it was saving just one of them, either generic **or** query specific

Reply:
I see 2 possible solutions to this:
1. make the selection of which to choose more explicit (e.g. using a function parameter) so that it is not only dependent on the number of streams.
2. save the 2 into different files.

Reply:
IMO:

1. Scrape netdata according to settings, i.e. get all timelines for all selected charts & save wherever you like
2. If it is a single stream, process wrt statistical functions, save as secondary file